src/main/java/cd/go/contrib/elasticagent/KubernetesSettings.java

```diff
@@ -0,0 +1,157 @@
+/*
+ * Copyright 2017 ThoughtWorks, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package cd.go.contrib.elasticagent;
+
+import org.apache.commons.lang3.StringUtils;
+import org.joda.time.Period;
+
+public class KubernetesSettings {
```

Comment:
How is this different from `PluginSettings` model?

Reply:
Good catch. This is not needed anymore. I have deleted it.